rclcpp/include/rclcpp/duration.hpp

```diff
@@ -143,6 +148,8 @@ class RCLCPP_PUBLIC Duration
 
 private:
   rcl_duration_t rcl_duration_;
+
+  Duration() = default;
```

Comment:
Why is this needed?

Reply:
To implement `Duration::from_seconds`, `Duration::from_nanoseconds` without reusing the deprecated constructor (see [here](https://github.com/ros2/rclcpp/blob/64f1a3e4a3585023b8959a92bb3a58bbc4640244/rclcpp/src/rclcpp/duration.cpp#L249-L263)).
I could also make use of `Duration::Duration(std::chrono::nanoseconds)` instead of adding the private default constructor.

Reply:
I see. I don't feel strongly about it.